Comp/Lipschitz.lean

```diff
@@ -4,12 +4,13 @@ import Comp.Defs
 # Lipschitz computations
 -/
 
-variable {ι I α : Type}
-variable {ω : ι → Type}
+variable {I α : Type}
+variable {ι : Type} {ω : ι → Type}
 variable {s : Set I}
 
 /-- An oracle computation is k-Lipschitz if the probabilities differ by `≤ k * oracle dist`.
     We define this asymmetrically, as we want the neighborhood of a particular oracle. -/
-structure Comp.lipschitz (f : Comp ι ω s α) (o : Oracle ι ω) (k : ℝ) : Prop where
+structure Comp.lipschitz (f : Comp (fun _ => ι) ω s α) (o : Oracle ι ω) (k : ℝ) : Prop where
```

Comment:
I don't understand this one enough to generalize, but really I only cared about generalizing `Defs` an d `Basic` anyway, and not breaking anything downstream.